### ISSUE TITLE
Put #include <tmmintrin.h> under PENGUINV_SSSE3_SET macro

### ISSUE THREAD

**Issue by ihhub** [ihhub/penguinV#443]
It's simple as header says we have to put `#include <tmmintrin.h>` under **PENGUINV_SSSE3_SET** macro in `src/image_function_simd.cpp` file.

**Comment by theoniko** [comment]
In file cpu_identification.h PENGUINV_SSSE3_SET will throw an error if PENGUINV_SSE_SET is not defined
and in the file image_function_simd.cpp when PENGUINV_SSE_SET is defined  header<tmmintrin.h> is included. 
For more details please refer to the code from cpu_identification.h and image_function_simd.cpp :
`#ifdef PENGUINV_SSSE3_SET
    #ifndef PENGUINV_SSE_SET
        #error "SSSE3 technology cannot exist without SSE. Please check SIMD instruction set verification code"
    #endif
#endif`

`#ifdef PENGUINV_SSE_SET
#include <emmintrin.h>
#include <tmmintrin.h>
#endif`

To sum up, in my opinion the requested header is anyway included and there is no need to change anything in the code.

**Comment by ihhub** [comment]
PENGUINV_SSE_SET might include SSSE3 or might not as SSSE3 is a subset of SSE. So if SSE is supported by compiler there is no guarantee that SSSE3 is supported to while the opposite always true.

**Comment by theoniko** [comment]
Hello @ihhub,

Ok i understand your point and I think that adding  the following code right after PENGUINV_SSE_SET macro mentioned above in file image_function_simd.cpp would be enought for this issue.

#ifdef PENGUINV_SSSE3_SET
#include <tmmintrin.h>
#endif

Please let me know if everything is fine so as to open a pull request.

**Comment by ihhub** [comment]
Hi @theoniko this is exactly what we need :)
